diff --git a/share/install-dependency b/share/install-dependency
index ab36ee4..072feb2 100755
--- a/share/install-dependency
+++ b/share/install-dependency
@@ -910,9 +910,9 @@ while [ $# -gt 0 ] ; do
 			SHA="dd45a55419026c592f8b6fc848dceface7e1ce98720bf13848a2e8ae366b29e8"
 			;;
 		wine-silverlight5.1-installer) # http://www.microsoft.com/getsilverlight/locale/en-us/html/Microsoft%20Silverlight%20Release%20History.htm
-			INS=(install_silverlight "5.1.41212.0" "latest")
-			URL="http://download.microsoft.com/download/5/3/D/53D3880B-25F8-4714-A4AC-E463A492F96E/41212.00/Silverlight.exe"
-			SHA="af2d435a034e03c2940a648291013ebd9de20d855447dbdb93416e7e03c3e91b"
+			INS=(install_silverlight "5.1.50907.0" "latest")
+			URL="https://download.microsoft.com/download/F/D/0/FD0B0093-DE8A-4C4E-BDC4-F0C56D72018C/50907.00/Silverlight.exe"
+			SHA="88e1b76bdf799478a72fa27db0bfe7bc5d02cc7e53675967399300448f0e266f"
 			;;
 		wine-flash-installer) # http://www.adobe.com/de/software/flash/about/
 			INS=(install_flash "23_0_0_205")
